src/components/menu/EditMenu.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import {removeSelectedEntity, cloneSelectedEntity} from '../../actions/entity';
+
+var menuOptions = {
+  'Clone': {group: 'base', callback: cloneSelectedEntity, needsEntity: true},
+  'Remove': {group: 'base', callback: removeSelectedEntity, needsEntity: true}
+};
+
+export const EditMenu = () => {
+  var prevGroup = null;
+  return (
+    <div className="menu">
+      <div className="title">Edit</div>
+      <div className="options">
+      {
+        Object.keys(menuOptions).map(function(key) {
```

Comment:
can do `key =>` for binding

lint should raise a couple things for this file too